config.yaml

```diff
@@ -110,6 +110,7 @@ params:
       job: Commissioner, City of Grand Rapids
       img: 4
   section5: true
+  section6: true
```

Comment:
This tells the `hugo-first` theme to look for `layouts/section6.html`

Reply:
got it.